boringtun/src/noise/handshake.rs

```diff
@@ -211,12 +211,13 @@ impl std::fmt::Debug for NoiseParams {
     }
 }
 
-struct HandshakeInitSentState {
+#[derive(Clone)]
+pub struct HandshakeInitSentState {
     local_index: u32,
     hash: [u8; KEY_LEN],
     chaining_key: [u8; KEY_LEN],
     ephemeral_private: x25519_dalek::ReusableSecret,
-    time_sent: Instant,
+    pub time_sent: Instant,
```

Comment:
`Instant` is technically an opaque, monotonic counter, and probably shouldn't be exposed like this.